src/doc/yaml/enrollment.yaml

```diff
@@ -0,0 +1,267 @@
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+        - name: attributes
+          in: query
+          description: The (optional) set of required attributes to retrieve
+          required: false
+          schema:
+            type: array
+            items:
+              type: string
+            example:
+              - surname
+              - familyname
+      responses:
+        '200':
+          description: Read successful
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Enrollment'
+        '400':
+          description: Bad request
+        '403':
+          description: Read not allowed
+        '404':
+          description: Unknown record
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    put:
+      tags:
+        - enrollment
+      summary: Update one enrollment
+      operationId: updateEnrollment
+      parameters:
+        - name: enrollmentId
+          in: path
+          description: the id of the enrollment
+          required: true
+          schema:
+            type: string
+        - name: transactionId
+          in: query
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Enrollment'
+      responses:
+        '204':
+          description: Update successful
+        '400':
+          description: Bad request
+        '403':
+          description: Update not allowed
+        '404':
+          description: Unknown record
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    delete:
+      tags:
+        - Enrollment
+      summary: Delete an enrollment
+      operationId: deleteEnrollment
+      parameters:
+        - name: enrollmentId
+          in: path
+          description: the id of the enrollment
+          required: true
+          schema:
+            type: string
+        - name: transactionId
+          in: query
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+      responses:
+        '204':
+          description: Delete successful
+        '400':
+          description: Bad request
+        '403':
+          description: Delete not allowed
+        '404':
+          description: Unknown record
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+  /v1/enrollments:
+    get:
```

Comment:
Passing a list of expressions as query parameters might be cumbersome. I propose to pass this list in the body, as done for [DataAccess/verify](https://osia.readthedocs.io/en/latest/annexes/technical/dataaccess.html#post--v1-persons-uin-verify). This means we would have to use a POST (and not a GET).

Reply:
With regards to the expression in body. Was it said how they are evaluated in case of multiple expressions?  Logical AND or OR?

Reply:
> With regards to the expression in body. Was it said how they are evaluated in case of multiple expressions? Logical AND or OR?
Good question. I would think maybe we could use regular expression?